django/applications/catmaid/control/nat/r.py

```diff
@@ -123,13 +123,13 @@ def export_skeleton_as_nrrd_async(skeleton_id, source_ref, target_ref, user_id,
                                   mirror=True, create_message=True) -> str:
 
     result = export_skeleton_as_nrrd(project_id, skeleton_id, source_ref,
-            target_ref, user_id, mirror)
+            target_ref, user_id, mirror) # FIXME: this function doesn't get project_id
```

Comment:
Guessing the method signature may need to be changed?

Reply:
Oops, that's true. It should get one though. I'll fix this after I merge this PR.